p2p/dht/dht_test.go

```diff
@@ -327,7 +327,7 @@ func bootAndWait(t *testing.T, dht DHT, errchan chan error) {
 // A bigger bootstrap
 func TestDHT_Bootstrap(t *testing.T) {
 
-	const timeout = 10 * time.Second
+	const timeout = 30 * time.Second
```

Comment:
What is the explanation for the tests taking more time on a local machine? what is the bottle neck? I'd refrain from extending timeout without knowing the root cause because that only means that we'll just keep updating them blindly and possibly hide bugs or deadlocks

Reply:
We're sending on multiple channels and running concurrent process that simulates a lot of nodes on a single machine. those channels might block for a while hence the test might take longer than 10 seconds. though on a fast machine (eg my mbp) it takes about 6 second. anyway I get you note and instead of that I'll just remove the timeout for that test.